examples/returns-from-sap/main.bal

```diff
@@ -0,0 +1,91 @@
+import ballerina/io;
+import ballerina/time;
+import ballerinax/googleapis.gmail as gmail;
+import osa/api_customer_returns_delivery_srv_0002 as returnDel;
+
+configurable string hostname = ?;
+configurable string username = ?;
+configurable string password = ?;
+
+configurable string refreshToken = ?;
+configurable string clientId = ?;
+configurable string clientSecret = ?;
+
+configurable string recipientAddress = ?;
+configurable string fromAddress = ?;
+
+returnDel:ConnectionConfig config = {
+    auth: {
+        username: username,
+        password: password
+    }
+};
+
+gmail:Client gmail = check new gmail:Client(
+    config = {
+        auth: {
+            refreshToken,
+            clientId,
+            clientSecret
+        }
+    }
+);
+
+public function main() returns error? {
+
+    returnDel:Client returnDelClient = check new (config, hostname);
+    io:println("Client is created successfully");
+    string query = "CreationDate gt datetime'" + getUTC() + "'";
+
+    returnDel:CollectionOfA_ReturnsDeliveryItemWrapper returnsDeliveries = check returnDelClient->listA_ReturnsDeliveryItems(
+        \$filter = query
```

Comment:
Give this inline

Reply:
Updated in [Add reviewed changes and documentation](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.delivery/pull/15/commits/918d0df8497a939d2833b9ed1b4509b83f1f89c0)